spec/rails_helper.rb

```diff
@@ -61,5 +61,8 @@
     with.test_framework :rspec
     with.library :rails
   end
+end
 
+RSpec.configure do |config|
+  config.include Rails.application.routes.url_helpers
```

Comment:
don't think i've seen this in a rails helper before, did you find that it was necessary to do things like `visit new_user_path`?